src/core/ext/filters/client_channel/lb_policy/pick_first/pick_first.cc

```diff
@@ -307,16 +309,16 @@ static void pf_update_locked(grpc_exec_ctx* exec_ctx, grpc_lb_policy* policy,
         grpc_lb_subchannel_list_ref_for_connectivity_watch(
             subchannel_list, "connectivity_watch+replace_selected");
         grpc_lb_subchannel_data_start_connectivity_watch(exec_ctx, sd);
-        if (p->subchannel_list != NULL) {
```

Comment:
Line 310 to 314 is moved down to fix a use-after-free (subchannel_list) bug. 

I am not sure why this PR can trigger such a bug by `if (p->selected->connected_subchannel != NULL)` while master is fine. Anyways, reordering works.